tests/Tests/400_ods_dashboard/400_ods_dashboard.robot

```diff
@@ -1,45 +1,61 @@
 *** Settings ***
-Resource         ../../Resources/ODS.robot
-Resource        ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
-Test Setup      Dashboard Test Setup
-Test Teardown   Dashboard Test Teardown
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.resource
 
+Test Setup          Dashboard Test Setup
+Test Teardown       Dashboard Test Teardown
 
 *** Test Cases ***
 Verify Resource Link Http status code
```

Comment:
Not sure if you've run robocop, but you should keep the formatting of TC/Keyword names consistent with `Title Case`, i.e. the first letter of each word is capitalized:
`Verify Resource Link Http Status Code`
I would also use `HTTP` since it is an acronym and usually written in that way.